rhobs/alerting/prometheus.pod_alerts.yaml

```diff
@@ -15,16 +15,24 @@ spec:
       labels:
         severity: warning
       annotations:
-        summary: "Pod {{ $labels.pod }} cannot be scheduled."
-        description: "Pod {{ $labels.pod }} for namespace {{ $labels.namespace }} on cluster {{ $labels.source_cluster }} is unscheduled for more than 30 minutes."
+        summary: Pod {{ $labels.pod }} cannot be scheduled.
+        description: >-
+          Pod {{ $labels.pod }} for namespace {{ $labels.namespace }} on cluster
+          {{ $labels.source_cluster }} is unscheduled for more than 30 minutes.
+          Related SOP:
+          https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-unschedualablePods.md
     - alert: CrashLoopBackOff
       expr: max_over_time(kube_pod_container_status_waiting_reason{reason="CrashLoopBackOff", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}[5m]) >= 1
       for: 15m
       labels:
         severity: warning
       annotations:
-        summary: "Pod {{ $labels.pod }} is crash looping"
-        description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') on cluster {{ $labels.source_cluster }} for more than 15 minutes."
+        summary: Pod {{ $labels.pod }} is crash looping
+        description: >-
+          Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in
+          waiting state (reason: 'CrashLoopBackOff') on cluster
+          {{ $labels.source_cluster }} for more than 15 minutes. Related SOP:
+          https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-crashLoopBackOff.md
     - alert: PodsNotReady
```

Comment:
Don't we want to add the link to https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-PodsNotReady.md for this alert?

Reply:
PodsNotReady is the one below.
I actually got confused with that a couple of times when working on that, so have another look to see that I'm seeing it correctly.

Reply:
In any case, PodsNotReady is covered by RHTAPWATCH-348. We need to make sure that we're providing the correct URLs and provide them under the correct alert definition.

Reply:
I understand that `PodsNotReady` would be in other PR